docs/unit.md

```diff
@@ -4,24 +4,70 @@ When running unit tests you should isolate your code so much that it could run i
 ## CLI commands
 A set of CLI commands are included in after-work.js to execute different test runners and tools.
 
-### aw-test-coverage (node.js)
-`aw-test-coverage` executes **Istanbul** to instrument the code and run tests to create a report of the current test coverage.
+### aw-test-runner (node.js)
+`aw-test-runner` is the default test runner for running unit tests with **Mocha** (using node.js). Appending the `cover` argument triggers nyc to instrument the code before the tests are executed and makes it possible to get test coveragee.
 ```
-aw-test-coverage -- test/unit
+aw-test-runner [cover] ./test/unit
 ```
+
 ##### Arguments
-  * `-- "folder"`: The starting point to run code coverage upon.
-  * `-x "folder"`: Excludes this folder from the code coverage.
-  * `--dir "folder"`: Specifies the output folder for the report.
+  * `cover`: Instruments the code and reports the test coverage
+  * `-w`: Watch files for changes which triggers a re-run. (not suitable to use with `cover` since coverage will only be reported on test end.)
+  * `--opts <file>`: path for mocha options file
+  * `--debug`: The script 'breaks' on the very first line. The debug session is attached with URL: chrome://inspect. Also shows the exact command, including all arguments, that is executed.
 
-### aw-test-runner (node.js)
-`aw-test-runner` is the default test runner for running unit tests with **Mocha** (using node.js).
+#### Usage recommendations for Nyc together with babel
+If you are using babel as transpiler it's recommended to [set up the project](https://github.com/istanbuljs/nyc#use-with-babel-plugin-istanbul-for-babel-support) with the `babel-plugin-istanbul` and the `cross-env` npm modules.
+
+.babelrc
 ```
-aw-test-runner ./test/unit
+{
+  "babel": {
+    "presets": ["env"],
+    "env": {
+      "test": {
+        "plugins": ["istanbul"]
+      }
+    }
+  }
+}
 ```
 
-##### Arguments
-  * `-w`: Watch files for changes which triggers a re-run.
+package.json
+```
+"scripts": {
+  "test": "cross-env NODE_ENV=test aw-test-runner <tests>"
+}
+```
+
+#### Default Arguments added by after-work.js
+The runner appends some default arguments:
+
+**Default:**
+  * Mocha:
+    * --compilers js:babel-core/register
+    * --opts dist/config/mocha.opts  (= recursive)
+    * --require dist/config/global.js
+
+**Cover:**
+  * Nyc:
+    * --source-map false (if `babel-plugin-istanbul` is used)
+    * --instrument false (if `babel-plugin-istanbul` is used)
+    * --reporter=text
+    * --reporter=lcov
+    * --all
+    * --include src
+    * --require babel-register
+  * Mocha:
+    * --opts dist/config/mocha.opts (= recursive)
+    * --require dist/config/global.js
+
+#### Override nyc options
+If you want to adjust the arguments for [nyc](https://github.com/istanbuljs/nyc#configuring-nyc) you are recommended to create a .nycrc or add a 'nyc object' in package.json.
```

Comment:
Sometimes you refer to Nyc with a capital N and sometimes only lower-case. Should be consistent imo.